src/no-side-effects.spec.ts

```diff
@@ -0,0 +1,160 @@
+// no-side-effects.spec.ts
+
+/*
+ * Copyright (c) 2024 Check Digit, LLC
+ *
+ * This code is licensed under the MIT license (see LICENSE.txt for details).
+ */
+
+import { RuleTester } from '@typescript-eslint/rule-tester';
+import { describe } from '@jest/globals';
+
+import rule from './no-side-effects';
+
+describe('no-side-effects', () => {
+  const ruleTester = new RuleTester();
+  // @ts-expect-error, workaround for the type definition
```

Comment:
are we sure we need this?  seems concerning

Reply:
RuleTester imported from eslint doesn't parse the test code properly and fails but RuleTester from @typescript-eslint/rule-tester works as expected